torch/nn/modules/transformer.py

```diff
@@ -314,7 +314,7 @@ def forward(
                 compatibility.
 
         Shape:
-            see the docs in Transformer class.
+            see the docs in `Transformer <https://pytorch.org/docs/stable/generated/torch.nn.Transformer.html>` class.
```

Comment:
Could you try this please (similarly for below)

```suggestion
            see the docs in :class:`~torch.nn.Transformer`
```

Reply:
Modified all places accordingly.